dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue

```diff
@@ -2,38 +2,21 @@
   <v-card elevation="0" class="fill-height" color="transparent">
     <v-scroll-y-transition mode="out-in">
       <v-row class="pa-2 fill-height" justify="space-between">
-        <v-col
-          style="min-width: 100px; max-width: 100%"
-          class="flex-grow-1 flex-shrink-0"
-          cols="5"
-        >
+        <v-col style="min-width: 100px; max-width: 100%" class="flex-grow-1 flex-shrink-0" cols="5">
```

Comment:
keep it in multiple lines, is most easy to read the code

Reply:
done